bin/oref0-autotune.sh

```diff
@@ -153,7 +153,7 @@ else
     cp -up settings/pumpprofile.json settings/profile.json || die "Cannot copy settings/pumpprofile.json"
 fi
 # If a previous valid settings/autotune.json exists, use that; otherwise start from settings/profile.json
-cp settings/autotune.json autotune/profile.json && cat autotune/profile.json | json | grep -q start || cp autotune/profile.pump.json autotune/profile.json
+cp settings/autotune.json autotune/profile.json && cat autotune/profile.json | jq | grep -q start || cp autotune/profile.pump.json autotune/profile.json
```

Comment:
The syntax you want here is `jq .`

Reply:
isn't that the same as I proposed?  Did I miss something?  

I was about to look at the ns-get.sh next

Reply:
The `.` is important.

Reply:
doh,  saw that twice and twice i said "I wonder if that dot is meant to be left or right of the single backquote."  And twice I guessed wrong.  LOL